tests/end-to-end/log-manager/setup_suite.bash

```diff
@@ -8,14 +8,73 @@ setup_suite() {
   with_kubeconfig sc
   with_namespace fluentd-system
 
-  # Pause logs being flushed to object storage
-  fluentd_aggregator_replicas="$(kubectl -n "${NAMESPACE}" get statefulset fluentd-aggregator -o jsonpath='{.spec.replicas}')"
-  kubectl -n "${NAMESPACE}" scale statefulset fluentd-aggregator --replicas 0 --timeout 5m
+  # Temporarily allow all traffic
```

Comment:
For future readers this might be confusing, please add a why here. Something like

```suggestion
  # Temporarily allow all traffic to not cause dropped packets and fail the Network Policy test suite
```